pkgs/development/libraries/libnotify/default.nix

```diff
@@ -1,22 +1,34 @@
-{ stdenv, fetchurl, pkgconfig, autoreconfHook
-, glib, gdk_pixbuf, gobject-introspection }:
+{ stdenv, fetchurl, meson, ninja, pkgconfig
+, glib, gdk_pixbuf, gobject-introspection, gnome3 }:
 
 stdenv.mkDerivation rec {
-  ver_maj = "0.7";
-  ver_min = "7";
-  name = "libnotify-${ver_maj}.${ver_min}";
+  pname = "libnotify";
+  version = "0.7.8";
 
   src = fetchurl {
-    url = "mirror://gnome/sources/libnotify/${ver_maj}/${name}.tar.xz";
-    sha256 = "017wgq9n00hx39n0hm784zn18hl721hbaijda868cm96bcqwxd4w";
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1371csx0n92g60b5dmai4mmzdnx8081mc3kcgc6a0xipcq5rw839";
   };
 
-  # disable tests as we don't need to depend on gtk+(2/3)
-  configureFlags = [ "--disable-tests" ];
+  mesonFlags = [
+    # disable tests as we don't need to depend on gtk+(2/3)
+    "-Dtests=false"
+    "-Ddocbook_docs=disabled"
+    "-Dgtk_doc=false"
+  ];
 
-  nativeBuildInputs = [ pkgconfig autoreconfHook gobject-introspection ];
+  patches = stdenv.lib.optional stdenv.hostPlatform.isDarwin ./no-version-script.patch;
```

Comment:
can we fetchpatch from https://gitlab.gnome.org/GNOME/libnotify/commit/55eb69247fe2b479ea43311503042fc03bf4e67d.patch (with a comment linking to https://gitlab.gnome.org/GNOME/libnotify/merge_requests/9) instead?